feature/gribi/otg_tests/mpls_in_udp/README.md

```diff
@@ -1,153 +1,216 @@
+2.  **Negative Test**: Verify traffic fails when MPLS-in-UDP entries are
+    removed
+    - Delete the gRIBI entries in reverse order
+    - Send the same traffic
+    - Validate 100% packet loss (no forwarding path)
 
-  # Paths added for TE-18.1.1 Match and Encapsulate using gRIBI aft modify
+## Packet Validation Details
+
+The test performs detailed packet capture validation:
+
+1.  **Capture Analysis**: Uses OTG packet capture on port-2 to analyze
+    encapsulated packets
+2.  **Header Parsing**: Validates both gopacket parsing and manual
+    byte-level parsing of UDP headers
+3.  **MPLS Validation**: Decodes MPLS header from UDP payload and
+    validates label, BoS bit, and TTL
+4.  **Error Detection**: Identifies common issues like incorrect UDP
+    ports or malformed MPLS headers
+
+## Expected Behavior
+
+- **RFC 7510 Compliance**: DUT must use UDP destination port 6635 for
+  MPLS-in-UDP encapsulation
+- **Proper Encapsulation**: Inner IPv6 packets are encapsulated with
+  MPLS label 100 inside UDP
+- **Header Preservation**: DSCP and TTL values are correctly set in
+  outer headers
+- **Forwarding**: Encapsulated packets are properly routed to the egress
+  port
+
+## Config Parameter Coverage
+
+- network-instances/network-instance/name
+- network-instances/network-instance/policy-forwarding/policies/policy/policy-id
+- network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/sequence-id
+- network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/network-instance
+
+## Telemetry Parameter Coverage
+
+- network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/id
+- network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/state/index
+- network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/index
+- network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/type
+- network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/mpls/state/mpls-label-stack
+- network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/src-ip
+- network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/dst-ip
+- network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/src-udp-port
+- network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/dst-udp-port
+- network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/ip-ttl
+- network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/dscp
+
+## Protocol/RPC Parameter Coverage
+
+- gRIBI:
+  - Modify
+    - ModifyRequest: AFT entries for MPLS-in-UDP encapsulation
+  - Flush
+    - FlushRequest: Clean up AFT entries
```

Comment:
this can be removed,  it is redundant to the  `## OpenConfig Path and RPC Coverage` heading and content.